src/sdk/pynni/nni/_graph_utils.py

```diff
@@ -412,30 +424,42 @@ def _build_graph(self):
                 module_to_nodes[module_name].append(node)
             else:
                 func_to_nodes[node.scopeName()].append(node)
-
         # build node group for module
         for module_name, node_cpps in module_to_nodes.items():
-            node_group = self._build_module_node_group(
-                module_name, module_to_type[module_name], node_cpps, input_to_node, output_to_node
-            )
-            _logger.debug('node_group: %s', node_group)
-            nodes_py.nodes_op.append(node_group)
+            use_count = 0
+            for node in node_cpps:
+                if not node.kind().startswith('prim::'):
+                    # modules that have same scope name may have different locations in the
+                    # graph. Futhermore, there are also lots of prim:: nodes that in node_cpps,
+                    # so we also need to call the expand_non_prim_node.
+                    unique_name = module_name
+                    if use_count > 0:
```

Comment:
It seems use_count is applied on name except use_count == 0,  can we just always apply use_count in unique_name?

Reply:
I suggest adding the suffix only when usecount>1, because most models will not be reused. Therefore, in most cases, the model will retain his original scopename. In this case, the name of the layers like conv can still be the same as the name in the name_modules() function which is more convenient. We can schedule a quick meeting to discuss this.